VVDocumenter-Xcode/Setting/VVDSettingPanelWindowController.m

```diff
@@ -54,6 +56,11 @@ - (void)windowDidLoad
     self.btnUseSpaces.state = (NSCellStateValue)[[VVDocumenterSetting defaultSetting] useSpaces];
 
     self.btnAddSinceToComment.state = (NSCellStateValue)[[VVDocumenterSetting defaultSetting] addSinceToComments];
+    self.mtxSinceOptions.enabled = [[VVDocumenterSetting defaultSetting] addSinceToComments];
+    [self.mtxSinceOptions selectCellAtRow:(NSInteger)[[VVDocumenterSetting defaultSetting] sinceOption] column:0];
+    self.tfSinceVersion.enabled = [[VVDocumenterSetting defaultSetting] addSinceToComments];
+    self.tfSinceVersion.stringValue = [[VVDocumenterSetting defaultSetting] sinceVersion];
```

Comment:
This will cause crash if the sinceVersion is `nil` (initial installed or did not set before). Please add something like `?: @""` or a default value beside of `nil` for it.